src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodConfigurationService.scala

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import javax.ws.rs.Path
+
+import akka.actor.{Props, Actor}
+import com.wordnik.swagger.annotations._
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+import org.broadinstitute.dsde.firecloud.model.{WorkspaceName, Destination, MethodConfigurationCopy}
+import org.broadinstitute.dsde.firecloud.{FireCloudConfig, HttpClient}
+import org.slf4j.LoggerFactory
+import spray.client.pipelining._
+import spray.routing._
+import spray.httpx.SprayJsonSupport._
+
+class MethodConfigurationServiceActor extends Actor with MethodConfigurationService {
+  def actorRefFactory = context
+  def receive = runRoute(routes)
+}
+
+@Api(value = "/workspaces/{workspaceNamespace}/{workspaceName}/method_configs",
+  description = "Method Configuration Services",
+  produces = "application/json")
+trait MethodConfigurationService extends HttpService with FireCloudDirectives {
+
+  private final val ApiPrefix = "workspaces"
+  lazy val routes = copyMethodRepositoryConfigurationRoute
+  lazy val log = LoggerFactory.getLogger(getClass)
+
+  @Path(value = "/copyFromMethodRepo")
+  @ApiOperation(
+    value = "copy a method repository configuration to a workspace",
+    nickname = "copyMethodRepositoryConfigurationToWorkspace",
+    httpMethod = "POST",
+    notes = "Copy a Method Repository Configuration into a workspace")
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "workspaceNamespace", required = true, dataType = "string", paramType = "path", value = "Workspace Namespace"),
```

Comment:
Sorry ... just saw this. The customer is the UI and the CLI. This field is not provided as a json object - it is a path parameter.